### PR TITLE
Add team page

### PR DESCRIPTION
**Pull request by cmarkel5**

### CODE REVIEW

**Comment by cmarkel5** [comment]
Everyone needs to submit their bio to me so that I can update. If someone wants to send theirs around to the group as an example for the rest to follow that would be cool. I will send mine around this weekend if I don't hear from anyone to get it started.

File: app/assets/templates/team.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<div class="container">
+  <div class="row text-center">
+    <div class="col-sm-3">
+      <p><img src="assets/Gerry400.jpg" class="img-circle team"/></p>
+      <p class="lead "><strong>Gerry Pass</strong></p>
```
Comment (rgpass) [inline app/assets/templates/team.html]: Class name should be representative of what it's doing. Also, random space here.

File: app/assets/templates/team.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      <p>Senior Developer</p>
```
Comment (rgpass) [inline app/assets/templates/team.html]: Let's do "Lead Developer" and you guys are "Developer"

File: app/assets/templates/team.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    </div>
+    <div class="col-sm-3">
+      <p><img src="assets/Chris400.jpg" class="img-circle team"/></p>
+      <p class="lead"><strong>Chris Markel</strong></p>
+      <p>Junior Developer</p>
+    </div>
+    <div class="col-sm-3">
+      <p><img src="assets/Kevin400.jpg" class="img-circle team"/></p>
+      <p class="lead"><strong>Kevin Abdo</strong></p>
+      <p>Project Manager</p>
```
Comment (rgpass) [inline app/assets/templates/team.html]: KDawgs going for a Product Manager gig and it's probably more in line with what he's doing.

File: app/views/layouts/_navigation.html.erb [hunk 4]
```diff
@@ -8,10 +8,8 @@
     <a class="navbar-brand" href="/">MiddleOf.Us</a>
   </div>
   <div class="navbar-collapse collapse navbar-inverse-collapse">
-    <ul class="nav navbar-nav">
-      <li><a href="/">Home</a></li>
-    </ul>
```
Comment (rgpass) [inline app/views/layouts/_navigation.html.erb:13]: :+1:

**Comment by rgpass** [comment]
To speed up this process, Google around to other groups and see what their bios include. We can include that + add in our own funny stuff. This way we have some direction as to what we should talk about.

**Pull request closed without merging**